packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -220,13 +276,12 @@
       this={component}
       options={enrichedOptions}
       {autocomplete}
-      value={selectedValue}
+      value={componentValue()}
```

Comment:
This won't be properly reactive, so changes in `selectedValue` won't actually result in this necessarily be re-rendered with the new value being passed in. Can we do `value={selectedValue as any}` instead?

Reply:
Sadly this isn't valid Typescript, I'll see what I can do to get around it - a reactive statement casting it to something like `castedSelectedValue` will probably be the method